package.json

```diff
@@ -15,6 +15,7 @@
   },
   "homepage": "https://github.com/connext/vector#readme",
   "devDependencies": {
+    "@protobuf-ts/plugin": "^1.0.12",
```

Comment:
We shouldn't have these dependencies at the top level.

Reply:
moved it to server-node dir package.json.